src/microbe_stage/MicrobeColony.cs

```diff
@@ -41,6 +42,17 @@ public Microbe.MicrobeState State
     [JsonProperty]
     public Microbe Master { get; private set; }
 
+    /// <summary>
+    ///   Creates a colony for a microbe, with this microbe as the master,
```

Comment:
I think "given" in this case probably makes more sense as we are passing a microbe.
```suggestion
    ///   Creates a colony for a microbe, with the given microbe as the master,
```